games-roguelike/FTL-gog/FTL-gog-1.6.3.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop unpacker
+
+DESCRIPTION="Top-down roguelike space ship simulator"
+HOMEPAGE="https://www.gog.com/game/faster_than_light"
+SRC_URI="ftl_advanced_edition_en_1_6_3_17917.sh"
+RESTRICT="bindist fetch mirror strip test"
+
+LICENSE="GOG-EULA"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+IUSE=""
+
+DEPEND="app-arch/unzip"
+RDEPEND="media-libs/freetype
+	media-libs/libsdl[X,sound,opengl,video]
+	media-libs/libpng
+	sys-libs/zlib
+	virtual/opengl"
+
+S="${WORKDIR}/data/noarch"
+
+pkg_nofetch() {
+	elog "Please buy and download ${SRC_URI} from"
+	elog "https://www.gog.com/game/faster_than_light"
+	elog "and copy it to ${DISTDIR}"
+}
+
+src_unpack() {
+	unpack_zip "${DISTDIR}/${A}"
+}
+
+src_prepare() {
+	default
+	rm -r support/{xdg*,*.{sh,txt}}
+	if ! use x86; then
+		rm game/data/FTL.x86 || die
+	fi
+	if ! use amd64; then
+		rm game/data/FTL.amd64 || die
+	fi
```

Comment:
I think this can be simplified to:
```bash
use x86 || rm game/data/FTL.x86 || die
use amd64 || rm game/data/FTL.amd64 || die
```

Reply:
Had issues with that syntax which I attempted first. Going to try it again to confirm. Even tried using `[ expr ]` syntax and had problems.

Reply:
@diogocp, we don't do double `||`, it's unreadable and unpredictable.